code/modules/mob/living/blood.dm

```diff
@@ -58,16 +62,22 @@
 					var/word = pick("dizzy","woozy","faint")
```

Comment:
you forgot this part

Reply:
it'll still say "You feel very dizzy/woozy/faint"